cmd/router/main.go

```diff
@@ -81,7 +82,11 @@ var runCmd = &cobra.Command{
 			return err
 		}
 
-		if rcfg.Daemonize || daemonize {
+		if console && daemonize {
```

Comment:
Why not (rcfg.Daemonize || daemonize) ?

Reply:
Because cli options are stronger than config pptions